hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/InvalidPathClientResponse.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.om.response;
+
+import org.apache.hadoop.hdds.utils.db.BatchOperation;
+import org.apache.hadoop.ozone.om.OMMetadataManager;
+import org.apache.hadoop.ozone.protocol.proto.OzoneManagerProtocolProtos;
+
+import javax.annotation.Nonnull;
+import java.io.IOException;
+
+/**
+ * This Response is only for catching InvalidPathException.
+ * It will clean cache in all tables.
+ */
+@CleanupTableInfo(cleanupAll = true)
```

Comment:
Can we specify a more specific table list to cleanup ?

Reply:
Since `InvalidPathClientResponse` not being used now, removed the class.